app/src/main/java/org/wikipedia/page/ExtendedBottomSheetDialogFragment.java

```diff
@@ -26,6 +31,23 @@ public void onCreate(Bundle savedInstanceState) {
     public void onStart() {
         super.onStart();
         setWindowLayout();
+        setPeekHeightPerDevice();
+    }
+
+    private void setPeekHeightPerDevice() {
```

Comment:
If you look in `AddToReadingListDialog` line 120, it shows you how to do this using a single line of code. Also let's do this in `ImagePreviewDialog`, instead of the base class.